config/initializers/active_storage_administrate_field.rb

```diff
@@ -0,0 +1,15 @@
+# as draw_routes is set to false in production, we need to override the calls to the url and blob_url methods
```

Comment:
C'est pas ça la raison, c'est plutôt : 
```
Since we don't want to host publicly accessible `/active_storage` routes for our assets, we override these methods to directly use the Scaleway URL instead 
```

Reply:
Mmm, je ne suis pas sûr : je suis d'accord que c'est parce que `we don't want to host publicly accessible "/active_storage" routes for our assets` que `draw_routes is set to false in production`, mais c'est parce que `draw_routes is set to false in production` et que active_storage + administrate utilise ces helpers que l'on a besoin de faire ce monkey_patch non ? J'ai un peu modifié le com

Reply:
Oui effectivement 👍